api/tests/routes/pro/post_offerer_invite_member_again_test.py

```diff
@@ -0,0 +1,88 @@
+import pytest
+
+import pcapi.core.offerers.factories as offerers_factories
+import pcapi.core.offerers.models as offerers_models
+import pcapi.core.users.factories as users_factories
+
+
+@pytest.mark.usefixtures("db_session")
+class Returns200Test:
+    def test_post_invite_member_again(self, client):
+        pro_user = users_factories.ProFactory(email="pro.user@example.com")
+        offerer = offerers_factories.OffererFactory()
+        offerers_factories.UserOffererFactory(user=pro_user, offerer=offerer)
+        offerers_factories.OffererInvitationFactory(user=pro_user, offerer=offerer, email="new.user@example.com")
+
+        data = {"email": "new.user@example.com"}
+
+        response = client.with_session_auth("pro.user@example.com").post(
+            f"/offerers/{offerer.id}/invite-again", json=data
+        )
+
+        assert response.status_code == 204
+        assert offerers_models.OffererInvitation.query.one()
+
+
+@pytest.mark.usefixtures("db_session")
+class Returns400Test:
+    def test_inivitation_does_not_already_exists(self, client):
```

Comment:
```suggestion
    def test_invitation_does_not_already_exist(self, client):
```
et les autres `inivitation`